containers/datalab/Dockerfile.in

```diff
@@ -116,8 +116,7 @@ RUN pip install protobuf==3.0.0b2.post2 && \
     rm tensorflow-0.8.0-cp27-none-linux_x86_64.whl
```

Comment:
We probably don't even need that line anymore. With the more flexible auth story we have now there is no real need to lock to a version.